tests/unit/test_converter_helpers.py

```diff
@@ -114,6 +114,46 @@ def test_get_entity_content_type_1():
     assert actual[0].mention_text == "411 I.T. Group"
 
 
+def test_get_entity_content_no_page_number():
+    docproto = documentai.Document()
+    page = documentai.Document.Page()
+    dimensions = documentai.Document.Page.Dimension()
+    dimensions.width = 2550
+    dimensions.height = 3300
+    page.dimension = dimensions
+    docproto.pages = [page]
```

Comment:
Can you change these object initializations to provide all of the input values at the start, instead of setting them later?
```suggestion
    docproto = documentai.Document(
        pages=[
            documentai.Document.Page(
                dimension=documentai.Document.Page.Dimension(
                    width=2550,
                    height=3300
                )
            )
        ]
    )
```

Reply:
Also, is there a reason this was added into this PR?

Not sure I see where it's relevant to the document revision stuff.

Reply:
No idea, why this was changed...